### DISCUSSION TITLE
Difference between RunPCA and RunSVD?

### DISCUSSION THREAD

**Discussion by agreco92** [satijalab/seurat#7097] [Q&A]
As per title, I noticed that most of the ATAC-related tutorials use the `RunSVD` function, while RNA data is processed using `RunPCA`. I remember that SVD solutions are also PCA solutions, i.e. the first eigenvectors are the ones that retain the most variance. Can someone explain me then what is the difference in principle between the two functions? I will also analyze the code on my own, but I'd like to understand the principle behind this. Thanks a lot! And thank you for this insanely good sc-omics software!

**Comment by wudustan** [comment]
Hi, from the Signac vignette:

The combined steps of TF-IDF followed by SVD are known as latent semantic indexing (LSI), and were first introduced for the analysis of scATAC-seq data by [Cusanovich et al. 2015](https://science.sciencemag.org/content/367/6473/45.full).

```
pbmc <- RunTFIDF(pbmc)
pbmc <- FindTopFeatures(pbmc, min.cutoff = 'q0')
pbmc <- RunSVD(pbmc)
```

This StackExchange [post](https://stats.stackexchange.com/questions/21311/when-to-choose-pca-vs-lsa-lsi) gives a summary on the difference
